### DISCUSSION TITLE
Date formats in i18n configuration

### DISCUSSION THREAD

**Discussion by asieira** [upptime/upptime#486] [Ideas]
Is there a way to change the date format? It's showing at DD/MM/YYYY for me as the default. 

It would be best to change the default to something unambiguous globally (i.e. `Dec 7, 2021` instead of `12/7/2021`) and/or allow the format to be overridden in the i18n configuration.

**Comment by sirchojin** [comment]
Hello,
I would also like to override date format, and also display 24H time format instead of AM/PM.

**Comment by AnandChowdhary** [comment]
Unfortunately this is currently not supported, but I'd love to accept a PR to add this feature. Here's how I'll go about it:

- Add a new key in the configuration for date/time format
- Use this key to localize .toLocaleDateString() in upptime/status-page
